scipy/_build_utils/tempita.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+
+from Cython import Tempita as tempita
+# XXX: If this import ever fails (does it really?), vendor either
```

Comment:
I don't think it ever fails anymore now that we do require Cython as a build dependency (wasn't always the case). It's just that it's not public API so Cython devs may decide to remove it at some point. But we can deal with it then.